src/tests/Numerical_aux/RegularizedYoffe.t.h

```diff
@@ -0,0 +1,76 @@
+#include "doctest.h"
+#include <Numerical_aux/RegularizedYoffe.h>
+#include <cassert>
+
+namespace seissol::unit_test {
+/**
+ * Yoffe function, see Tinti et al. 2005: eq 1
+ */
+real yoffe(real time, real riseTime) {
+  constexpr real epsilon = 10 * std::numeric_limits<real>::epsilon();
+  if (time < epsilon || time > riseTime - epsilon) {
+    return 0.0;
+  } else {
+    return 2 / (M_PI * riseTime) * std::sqrt((riseTime - time) / time);
+  }
+}
+
+/**
+ * Triangle function, see Tinti et al. 2005: eq 2
+ */
+real triangle(real time, real halfDuration) {
+  assert(halfDuration > 0);
+  real halfDurationSquared = halfDuration * halfDuration;
+  if (time > 0 && time < halfDuration) {
+    return time / halfDurationSquared;
+  } else if (time > halfDuration && time < 2 * halfDuration) {
+    return (2 * halfDuration - time) / halfDurationSquared;
+  } else {
+    return 0;
+  }
+}
+
+/**
+ * Reference implementation for the regularized Yoffe function by explicitly computing the
+ * convolution, see Tinti et al. 2005: eq 3
```

Comment:
Unfortunately, it is not applicable, because you need a really fine resolution to compute the integrals accurately. But I thought for testing, it is a good idea...